src/components/swap/SwapDetailsDropdown.tsx

```diff
@@ -131,6 +159,12 @@ export default function SwapDetailsDropdown({
   const theme = useTheme()
   const { chainId } = useWeb3React()
   const [showDetails, setShowDetails] = useState(false)
+  const [isFirstPriceFetch, setIsFirstPriceFetch] = useState(true)
+  const lpFeePercent = trade ? computeRealizedLPFeePercent(trade) : undefined
+
+  useEffect(() => {
```

Comment:
again can't think of a better way to do this right now but i do see we are mixing logging concerns with react state, which i think should be separate

Reply:
hmm yeah this was my best attempt at a decent way to capture this, can ask for @zzmp's input too when he's back.